data/src/test/java/com/linkedin/data/schema/resolver/TestDataSchemaResolver.java

```diff
@@ -236,6 +238,7 @@ public void testMapDataSchemaResolver()
     {
         {
           "Two records including each other",
+            "pdsc",
```

Comment:
Use `com.linkedin.data.schema.SchemaFormatType` here rather than a String. It's more type-safe.

This also gives you easy access to the parser factory instance.